pkl-core/src/main/java/org/pkl/core/Release.java

```diff
@@ -181,24 +116,32 @@ public String getFilePage(String path) {
       return homepage + "blob/" + version + "/" + path;
     }
```

Comment:
I didn't do this because I think the record naming convention should only be used for getters.
In particular the `myProperty(singleArg)` syntax should be reserved for setters. (In the case of records, a setter would return a new record.)

Reply:
In that case, `sourceCodeUrlScheme()` isn't a getter either. Should that change be reverted?

Reply:
I'd say `sourceCodeUrlScheme()` is fine because it returns (logical) state. But I wouldn't use this naming convention for methods that have parameters.

Reply:
Oh, I missed that! Yeah, this makes sense.